src/pages/ceph/users/index.vue

```diff
@@ -61,28 +82,106 @@ const { data: cephUsers } = useLazyAsyncData('cephUsers', async () => {
                                     <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
                                         Enabled
                                     </th>
-                                    <th scope="col" class="relative py-3.5 pl-3 pr-4 sm:pr-6">
-                                        <span class="sr-only">Edit</span>
+                                    <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
+                                        Delete
                                     </th>
                                 </tr>
                             </thead>
                             <tbody class="divide-y divide-gray-200 bg-white">
-                                <tr v-for="cephUser in cephUsers" :key="cephUser?.Username">
+                                <tr v-for="{ Enabled, Roles, Username } in cephUsers" :key="Username">
                                     <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6">
-                                        {{ cephUser.Username }}
+                                        {{ Username }}
                                     </td>
-                                    <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">{{ cephUser.Roles }}</td>
-                                    <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">{{ cephUser.Enabled }}</td>
-                                    <td
-                                        class="relative whitespace-nowrap py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-6"
-                                    >
-                                        <a href="#" class="text-indigo-600 hover:text-indigo-900"
-                                            >Edit<span class="sr-only">, something</span></a
-                                        >
+                                    <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">{{ Roles }}</td>
```

Comment:
This is how it might look when using the generic badge in a vue template loop:

```suggestion
                                    <td v-for="role in Roles" class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
                                        <GenericBadge>{{ role }}</GenericBadge>
                                    </td>
```

(Don't forget to import the `GenericBadge.vue` component; this assumes the `Roles` variable is an array with the roles)

Reply:
Awesome, it looks great
![image](https://github.com/koor-tech/data-control-center/assets/638376/b6fcceb4-24ef-4756-95fc-c1427f327aeb)